lampod/src/ln/inventory_manager.rs

```diff
@@ -33,11 +33,32 @@ impl InventoryHandler for LampoInventoryManager {
         match event {
             InventoryCommand::GetNodeInfo(chan) => {
                 let chain = self.channel_manager.conf.network.to_string();
+                let alias = self
+                    .channel_manager
+                    .conf
+                    .alias
+                    .as_ref()
+                    .map(|alias| alias.to_string());
+                // we have to put "" in case of alias missing as cln provide us with a random alias.
+                let alias = alias.unwrap_or_else(|| "".to_string());
+                let (_, height) = self.channel_manager.onchain.backend.get_best_block()?;
+                let blockheight;
+                if height.is_none() {
+                    blockheight = 0
+                } else {
+                    // Safe to unwrap here
+                    blockheight = height.unwrap();
+                }
```

Comment:
```suggestion
                let blockheight = height.unwrap_or_default();
```

Before, you used function chaining excessively, and now, scarcely :)

Reply:
Oops.